dynawo/sources/Modeler/Common/DYNModelMulti.h

```diff
@@ -492,7 +483,9 @@ class ModelMulti : public Model, private boost::noncopyable {
   int sizeMode_;  ///< number of mode
   int sizeY_;  ///< number of continuous values
   int sizeCalculatedVar_;  ///< number of calculated variables
-  bool zChange_;  ///< @b true if one discrete value has changer
+  bool zChange_;  ///< @b true if one discrete value has changed
+  bool modeChange_;  ///< @b true if one mode has changed
+  modeChangeType_t modeChangeType_;  ///< type of mode change (No_Change, Differential, Algebraic, Algebraic_J_Update)
```

Comment:
No need to put the possibilities of the enum as it might evolve (and this list would be outdated)

Reply:
Done